.github/workflows/linters.yaml

```diff
@@ -0,0 +1,101 @@
+name: Linters
+
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
I'd like to run tests also for release branches. Basically I want to run test for each PR

Reply:
OK. Same change for atomic-reactor?

Reply:
tldr; add 'release_*' to `on: pull_request` for linters and unittests

Reply:
Added